config_defaults.yml

```diff
@@ -27,9 +27,34 @@ Gemfile:
         version: '~>0.10.5'
       - gem: pry
     ':system_tests':
-      - gem: beaker-rspec
       - gem: beaker
+        from_env: 'BEAKER_VERSION'
+        version: '~> 2.20'
+        condition: "supports_windows"
+      - gem: beaker
+        from_env: 'BEAKER_VERSION'
+        condition: "Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.3.0') and ! supports_windows"
+      - gem: beaker
+        from_env: 'BEAKER_VERSION'
+        version: '< 3'
+        condition: "Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.3.0') and ! supports_windows"
+      # beaker 3 requires this for PE installs
+      - gem: beaker-pe
+        condition: "Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.3.0')"
+      - gem: beaker-rspec
+        from_env: 'BEAKER_RSPEC_VERSION'
+        # newer version required to avoid BKR-537
+        version: '>= 3.4'
+        condition: "! supports_windows"
+      - gem: beaker-rspec
+        from_env: 'BEAKER_RSPEC_VERSION'
+        # newer version required to avoid BKR-537
+        version: '~> 5.1'
+        condition: "supports_windows"
```

Comment:
Not because the gem conditions are odd, but because once master and windows branches are merged there will be no location to specify which modules support windows.

I don't particularly like the gem declaration condition, but the config_defaults.yml doesn't have meta-conditions of whether to add lines to the template or not, only explicit Gemfile conditions. I guess I could make Yet Another Key of `windows: true/false` that could specify whether a gem should be added to windows or non-windows gems, but I kind of what the distinction to go away anyway, not just bring a bunch of logic into a template that perpetuates the separation.

If can unpin from beaker-rspec 5.x or universally pin to it, that would be better overall.